gems/policy-parser/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2020 CyberArk Software Ltd. All rights reserved.
```

Comment:
Do we need to license the policy parser individually?

Reply:
As it's code that moves from one codebase to another, we should add the Parser dependencies to the license file.  The Policy Parser code now uses the Conjur license.